src/engraving/libmscore/excerpt.cpp

```diff
@@ -1470,10 +1474,7 @@ void Excerpt::cloneStaff2(Staff* srcStaff, Staff* dstStaff, const Fraction& star
                     }
 
                     for (EngravingItem* e : oseg->annotations()) {
-                        if (e->generated()) {
-                            continue;
-                        }
-                        if (e->track() != srcTrack) {
+                        if (e->generated() || e->track() != srcTrack || e->systemFlag()) {
```

Comment:
This change causes a regression of this issue: https://github.com/musescore/MuseScore/issues/11186

See the video:

https://user-images.githubusercontent.com/67867444/201324070-7585c16b-cf28-4b72-9bdd-2500c47c7e21.mp4

Reply:
Ha! Thanks, I'll see if I can find a different fix